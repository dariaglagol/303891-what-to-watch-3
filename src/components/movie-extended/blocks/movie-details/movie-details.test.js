import React from "react";
import renderer from "react-test-renderer";
import MovieDetails from "./movie-details";

const MovieDetailsMock = {
  description: `description`,
  director: `director`,
  starring: `starring`,
  score: 8.0,
  rating: 8.0,
};

it(`Render movie details`, () => {
  const {description, director, starring, score, rating} = MovieDetailsMock;
  const movieDetailsComponent = renderer
    .create(
        <MovieDetails
          description={description}
          director={director}
          starring={starring}
          score={score}
          rating={rating}
        />
    ).toJSON();

  expect(movieDetailsComponent).toMatchSnapshot();
});
