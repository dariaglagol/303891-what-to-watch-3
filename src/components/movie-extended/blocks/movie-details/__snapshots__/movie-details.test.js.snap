// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render movie details 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        8
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      description
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        starring
      </strong>
    </p>
  </div>,
]
`;
